### ISSUE TITLE
[BUG] Swipe breaks if connection error occurs during response generation

### ISSUE THREAD

**Issue by haowdi** [SillyTavern/SillyTavern#1216]
### Environment

Self-Hosted (Bare Metal)

### System

Firefox 118.0.1

### Version

1.10.4

### Describe the problem

Like the title says, if a connection error or any unknown error occurs during response generation, the swipe function breaks. The left swipe icon completely disappears and the right swipe icon won't work (clicking on it does nothing).
I tried creating a copy of the message - via the copy button next to the delete button - but the broken swipe carries over to the copy too. Then I tried closing and reloading the chat but the swipe for that message is still broken so I had to delete the message altogether...

I don't think this bug occurs for errors that SillyTavern recognizes, as in those with custom red pop-ups on the interface, because I've encountered them before (for the current SillyTavern  1.10.4) and the swipe function still worked then. So maybe it only happens to errors without pop-ups...?

To add, I think this swipe bug only started happening after the loop-around feature was added, because I encountered connection issues in previous SillyTavern versions but have never had this bug...until now. I could be wrong, sorry.

### Additional info

![swipe](https://github.com/SillyTavern/SillyTavern/assets/143865892/07846931-60cc-4ee5-99d4-d1c243a8a800)

^ Broken message without left swipe and unusable right swipe.

![cmd_GA4ofYjHdO](https://github.com/SillyTavern/SillyTavern/assets/143865892/9d5ec5ac-1d2c-4e3d-94df-bef08dee7dee)

^ Unknown error in cmd. I already encountered this broken swipe bug once before (for the current SillyTavern 1.10.4) where the cmd says "Connection error" or something akin to that. Forgot to screenshot it, sorry.

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You've checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
What causes the unknown error? Do you use the official endpoint with the API key?

**Comment by Cohee1207** [comment]
Most of the swipe UI state can be fixed by refreshing the page. Have you tried that?

**Comment by haowdi** [comment]
I was using a proxy (I know that's uh...not officially supported) but I feel like it's more of a swipe issue than a proxy issue.

Oh and I forgot to mention, I did refresh the page (even stopped and re-started SillyTavern) but the broken swipe persists.

I want to add another thing, I mentioned already encountering this bug once before (with "Connection error" and not unknown error) and refreshing the page did fix it. The only difference is last time I didn't mess around and try to create a copy of the message. This time I think I messed up by trying to create a copy and the swipe bug just...persists...
